src/backend/distributed/commands/dependencies.c

```diff
@@ -31,20 +31,65 @@
 #include "distributed/worker_manager.h"
 #include "distributed/worker_transaction.h"
 
+typedef enum RequiredObjectSet
+{
+	REQUIRE_ONLY_DEPENDENCIES = 1,
+	REQUIRE_OBJECT_AND_DEPENDENCIES = 2,
+} RequiredObjectSet;
+
```

Comment:
Naming can be quite tricky here :) First, we need a comment on this struct that explains clearly that this struct is created for function that ensures that objects exist on all nodes etc.
Here goes my naming suggestion:
```suggestion
typedef enum RequiredObjectSet
{
	ONLY_DEPENDENCIES = 1,
	DEPENDENCIES_AND_OBJECT = 2,
} ObjectsToExistOnAllNodes;

```

Reply:
Decided with @onurctirtir to keep as it is